packages/block-library/src/template-part/edit/utils/hooks.js

```diff
@@ -150,8 +150,12 @@ export function useTemplatePartArea( area ) {
 				).__experimentalGetDefaultTemplatePartAreas();
 			/* eslint-enable @wordpress/data-no-store-string-literals */
 
-			const selectedArea = find( definedAreas, { area } );
-			const defaultArea = find( definedAreas, { area: 'uncategorized' } );
+			const selectedArea = definedAreas.find(
```

Comment:
`definedAreas` is always an array:

https://github.com/WordPress/gutenberg/blob/8ac430b799abea21a3368000ca936eabf432bddd/packages/editor/src/store/selectors.js#L1561-L1565